tests/kernel/kernel_info.cpp

```diff
@@ -44,15 +45,72 @@ TEST_CASE("Test kernel info", "[kernel]") {
 
   /** check program info parameters
    */
-  auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
-  check_return_type<uint32_t>(
-      uint32Ret, "sycl::kernel::get_info<sycl::info::kernel::num_args>()");
+  auto incorrect_num_args_usage = [&kernel] {
+    auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
+  };
+
+  {
+    /** According to SYCL specification info::kernel::num_args descriptor may
+       only be used to query a kernel that resides in a kernel bundle that was
+       constructed using a backend specific interoperability function or to
+       query a device built-in kernel. In other cases an exception with the
+       errc::invalid error code is thrown. We attampt to use
```

Comment:
```suggestion
       errc::invalid error code is thrown. We attempt to use
```
Please use a spell-checker to avoid wasting time of the reviewers.

Reply:
Fixed. Installed spell-checker extension for vsc.